### ISSUE TITLE
Add new "category" field to canned response

### ISSUE THREAD

**Issue by EnTeQuAk** [mozilla/addons-server#11386]
For the code-manager and line-based comments, we need another categorization method in addition to being filtered by wished action.

We do have `sort_group` which matches the intended review-action (`public`, `reject`, `confirm_auto_approved` etc) but we need to store extra information like "Security", "Privacy", "Development practises" etc that can be filtered on easier for line-based comments.

**Comment by AlexandraMoga** [comment]
@EnTeQuAk I've tried to look at the new [canned-responses](https://addons-server.readthedocs.io/en/latest/topics/api/reviewers.html#get--api-v4-reviewers-canned-responses-) enpoint, but I'm receiving a 500 error with each request.
I've tried both on -dev and stage while being authenticated with a user with reviewer permissions.

**Comment by EnTeQuAk** [comment]
https://sentry.prod.mozaws.net/operations/olympia-dev/issues/5741459/?query=is:unresolved

I'll have a fix up in a bit.

**Comment by AlexandraMoga** [comment]
The canned-responses api is working on -dev.
The endpoint is accessible only to users with reviewer permissions. 

Here are all the fields exposed:

```
        "id": 29,
        "title": "Approved for Public",
        "response": "This version has been approved for the public.\r\n\r\nThank you!",
        "category": "Other"
```

@EnTeQuAk I reckon the fix from #11450 wasn't pushed to stage yet, but it might not be so urgent to deserve a cherry picking.
